network-primitives/variables.tf

```diff
@@ -32,6 +32,7 @@ variable "full-node-config" {
     reserved-only       = bool
     prune               = bool
     node-dsn-port       = number
+    domain-id           = number
```

Comment:
forgot to remove these ?

Reply:
thanks, good catch!

Reply:
i actually removed them, and my git work tree is clean. Need to check why it wasn't staged. :/